.clang-format

```diff
@@ -4,51 +4,93 @@ Language: Cpp
 BasedOnStyle: Google
 AccessModifierOffset: -4
 AlignAfterOpenBracket: Align
+AlignConsecutiveMacros: true
 AlignConsecutiveAssignments: true
 AlignConsecutiveDeclarations: false
-AlignEscapedNewlinesLeft: false
+AlignEscapedNewlines: Left
 AlignOperands: true
 AlignTrailingComments: true
+AllowAllArgumentsOnNextLine: false
+AllowAllConstructorInitializersOnNextLine: true
 AllowAllParametersOfDeclarationOnNextLine: false
-AllowShortBlocksOnASingleLine: false
+AllowShortBlocksOnASingleLine: true
```

Comment:
Isn't `AllowShortBlocksOnASingleLine` this the same as `AllowShortIfStatementsOnASingleLine`, but for other blocks like `while`?

Would it be more consistent to configure them the same?

Reply:
Probably